Makefile

```diff
@@ -18,12 +18,12 @@ icons:		## Generate icons from stensil (requires Inkscape)
 translate:	## Translate and update .po and .mo files (requires PyBabel)
 	$(MAKE) -C po
 
-model: gaphor/core/modeling/coremodel.py gaphor/UML/uml2.py	## Generate Python model files from Gaphor models (requires Black, MyPy)
+model: gaphor/core/modeling/coremodel.py gaphor/UML/uml.py	## Generate Python model files from Gaphor models (requires Black, MyPy)
 
-gaphor/core/modeling/coremodel.py: models/Core.gaphor models/Core.override utils/model/gen_uml.py utils/model/override.py utils/model/writer.py
-	utils/model/build_core.py && black $@ && mypy gaphor/core/modeling
+gaphor/core/modeling/coremodel.py: models/Core.gaphor models/Core.override gaphor/codegen/autocoder.py gaphor/codegen/override.py gaphor/codegen/writer.py
+	gaphor/codegen/codegen.py Core && black $@ && mypy gaphor/core/modeling
```

Comment:
I would use specific arguments for the model, overrides and output files.

Reply:
@amolenaar Thanks so much for the feedback, I guess I was trying to make things a little too tricky! :100: Please let me know if you like the updated version better.